multiversx_sdk_cli/cli_wallet.py

```diff
@@ -32,6 +33,9 @@
 
 WALLET_FORMATS_AND_ADDRESSES = [*WALLET_FORMATS, WALLET_FORMAT_ADDRESS_BECH32, WALLET_FORMAT_ADDRESS_HEX]
 
+MAX_ITERATIONS_FOR_GENERATING_WALLET = 100
+CURRENT_SHARDS = [0, 1, 2]
```

Comment:
Maybe correlate this with:

https://github.com/multiversx/mx-sdk-py-cli/blob/main/multiversx_sdk_cli/constants.py#L15


So that only one constant should be updated in case of such a change.

Reply:
Done